include/fat.h

```diff
@@ -1,26 +1,26 @@
 /*
-* fat.h
-*
-* definitions for the fat file system
-*/
+ * fat.h
+ *
+ * definitions for the fat file system
+ */
 
 #ifndef _FAT_H
 #define _FAT_H
 
+#include "list.h"
 #include <stdint.h>
 #include <time.h>
-#include "list.h"
 
-#define FAT_MAX_CACHE_SECTOR    4
+#define FAT_MAX_CACHE_SECTOR 4
 
-#define FAT_ATTRIBUTE_DEFAULT    0x00
-#define FAT_ATTRIBUTE_READONLY   0x01
-#define FAT_ATTRIBUTE_HIDDEN     0x02
-#define FAT_ATTRIBUTE_SYSTEM     0x04
-#define FAT_ATTRIBUTE_LABEL      0x08
-#define FAT_ATTRIBUTE_DIRECTORY  0x10
-#define FAT_ATTRIBUTE_ARCHIVE    0x20
-#define FAT_ATTRIBUTE_DEVICE     0x40
```

Comment:
This seems to have gotten uglier. Maybe set `AlignConsecutiveMacros `? https://clang.llvm.org/docs/ClangFormatStyleOptions.html#alignconsecutivemacros

I'm not sure which of Consecutive/AcrossEmptyLines/AcrossComments/AcrossEmptyLinesAndComments would be best (probably empty lines at least for sd.h?)

Reply:
I think AcrossEmptyLines would be better than `true` in sd.h, and maybe AcrossEmptyLinesAndComments too (needs experimenting)

Reply:
I think AcrossEmptyLinesAndComments is probably best

Reply:
let me know if it needs to be changed again